azure-pipelines.yml

```
@@ -59,7 +59,8 @@ jobs:
       artifactName: test-signing
       targetPath: $(Build.StagingDirectory)/test-signing
   - bash: |
-      azuresigntool sign \
+      "/C/Program Files/dotnet/dotnet.exe" tool install --global AzureSignTool
+      /c/Users/u/.dotnet/tools/azuresigntool sign \
```

```suggestion
      /C/Users/u/.dotnet/tools/azuresigntool sign \
```
doesn’t matter but let’s be consistent between the two lines at least

---

👍